.travis.yml

```diff
@@ -1,5 +1,12 @@
+sudo: false
 language: ruby
 rvm:
   - 2.3.1
+before_install:
+  - gem install bundler -v 1.13.2
+  - gem install rake -v 10.0
+  - gem install oj -v 2.17.5
+  - gem install rspec -v 3.5.0
```

Comment:
To save info about gem versions, now I understand that it doesn't have any sense :(